tests/trainer/logging_/test_logger_connector.py

```diff
@@ -387,6 +387,7 @@ def _assert_called(model, stage):
         max_epochs=1,
         progress_bar_refresh_rate=0,
         num_sanity_val_steps=2,
+        checkpoint_callback=False,
```

Comment:
would setting it to True here be bad, i.e., the default? it wouldn't be due to the changes of this pr, right?

Reply:
Done !